packages/elements/src/slider/index.ts

```diff
@@ -598,18 +598,27 @@ export class Slider extends ControlElement {
       return;
     }
 
+    this.changedThumb = event.target as HTMLDivElement;
+    const thumbName = this.changedThumb.getAttribute('name') as SliderDataName;
```

Comment:
Can you detect the thumb name in different name, e.g. if you hold the reference to thumb:
```
private getThumbName (target: EventTarget): SliderDataName | null {
  if (target === this.thumbFromRef.value) {
     return ...
  }
  return ....
}
```